CONTRIBUTING.md

```diff
@@ -1,12 +1,12 @@
-# Contributing to Eclipse Project for JAX-RS
+# Contributing to the Jakarta RESTful Web Services Project
```

Comment:
I wonder why we not simply adopt #779 instead?

Reply:
This is #779. If covered by another PR, will drop it from here.

Reply:
You are right, I just noticed that a minute ago. Sorry for the confusion.

Can you please cherry-pick 66ce6b3b9ea61daab64153789bababc2c14a3d84 and rebase ontop instead of copying the content? That would be fair. :-)

Reply:
Does that mean you'll approve it if I do all that work? :-)

Reply:
"all that work"? You simply need to `git rebase -i HEAD^` and add `p 66ce6b3` in the first line and the `git push -f` to have a fair commit history. Should be done in one minute.

As I said, I am not +1 for removing the word "JAX-RS", but I do not stand in the way of a majority. So my vote will be "0".

Reply:
It was done, and I was joking.

Reply:
Thanks! :-) :tada:

Reply:
I am a bit disappointed that you sqashed the changes for the merge, after you agreed to explicitly keep my original commit standalone.